Formula/mas.rb

```diff
@@ -11,14 +11,31 @@ class Mas < Formula
     sha256 "af5be6aa9902d9cfc2aa69dbf313441a7c201463d516face721f900ceae9556b" => :sierra
   end
 
-  depends_on :xcode => ["9.0", :build]
+  depends_on :xcode => ["9.3", :build]
+
+  resource "cocoapods" do
+    url "https://dl.bintray.com/phatblat/mas-bottles/master.tar.gz"
```

Comment:
I don't think downloading an unversioned, non-official copy of the repo is acceptable in the Homebrew way of handling dependencies.

Reply:
> We have a `cocoapods` formula which we could depend on, couldn't we?

The `cocoapods` formula installs the command, not the repo.

Reply:
Understood. I think the only option is thus to wait for https://github.com/mas-cli/mas/pull/169 to be merged, which moves away from cocoapods and will allow a build compatible with Homebrew's rules.

Reply:
Wasn't this suggestion https://github.com/Homebrew/homebrew-core/pull/31058#issuecomment-413746161 using the official repo ?